src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -155,10 +143,6 @@ export function GraphWrapper({
 	onSelectionChange,
 	onDismissBanner,
 }: GraphWrapperProps) {
-	// TODO: application shouldn't know about the graph component's header
-	const graphHeaderOffset = 24;
-	const [mainWidth, setMainWidth] = useState<number>();
-	const [mainHeight, setMainHeight] = useState<number>();
 	const mainRef = useRef<HTMLElement>(null);
```

Comment:
`mainRef` was only needed for the `ResizeObserver`. This can be remove too.

Reply:
Done in commit: `Removed unneeded "mainRef" variable`.